src/main/java/nl/tudelft/lifetiles/graph/view/VertexView.java

```diff
@@ -42,9 +37,19 @@
     private double resizeWidth = -1;
 
     /**
-     * Spacing between rectangles.
+     * Horizontal and vertical spacing between rectangles.
+     */
+    private static final double SPACING = 3;
+
+    /**
+     * Horizontal scale for each coordinate.
      */
-    private final double spacingX = 3;
+    private static final double HORIZONTALSCALE = 15;
```

Comment:
I think so too, but we shouldn't store the spacing and the scale of the vertices in all the vertices. It would be better if we would centralize the variable in another class, the TileView for example.